src/Microsoft.AspNetCore.Identity.UI/IdentityBuilderUserInterfaceExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Mvc.Razor;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Identity
+{
+    public static class IdentityBuilderUserInterfaceExtensions
```

Comment:
Look in this class as a starting point

Reply:
I think we should just use UI everywhere as opposed to spelling it out UserInterface in some places and UI in others.